evals/benchmark/README.md

```diff
@@ -12,12 +12,171 @@ This Tool provides a microservice benchmarking framework that uses YAML configur
 
 ## Table of Contents
 
+- [Workload Deployment in Kubernetes](#workload-deployment-in-kubernetes)
+  - [Prerequisites](#prerequisites)
+  - [Deployment](#deployment)
+    - [1. Deploy with CPUs only](#1-deploy-with-cpus-only)
+    - [2. Deploy with Gaudi Devices](#2-deploy-with-gaudi-devices)
+    - [3. Deploy with Multiple Instances](#3-deploy-with-multiple-instances)
+    - [4. Deploy with a Specific Namespace](#4-deploy-with-a-specific-namespace)
+    - [5. Deploy with Proxy Support](#3-deploy-with-proxy-support)
+  - [Destroy](#destroy)
 - [Installation](#installation)
 - [Usage](#usage)
 - [Configuration](#configuration)
   - [Test Suite Configuration](#test-suite-configuration)
   - [Test Cases](#test-cases)
 
+## Workload Deployment in Kubernetes
+
+### Prerequisites
+
+- Kubernetes installation: Use [kubespray](https://github.com/opea-project/docs/blob/main/guide/installation/k8s_install/k8s_install_kubespray.md) or other official Kubernetes installation guides.
+- Helm installation: Follow the [Helm documentation](https://helm.sh/docs/intro/install/#helm) to install Helm.
+- Install the OPEA Helm Chart:
+
+  ```bash
+  helm repo add opea https://opea-project.github.io/GenAIInfra
+  ```
+- Setup Hugging Face Token
+  To access models and APIs from Hugging Face, set your token as environment variable.
+  ```bash
+  export HFTOKEN="insert-your-huggingface-token-here"
+  ```
+
+### Deployment
+
+#### 1. Deploy with CPUs only
+
+This setup ensures that all services in the opea/chatqna workload run with CPU resources only.
+
+```bash
+# Get the list of available charts:
+helm search repo opea
+
+# Deploy the opea/chatqna chart
+helm install chatqna opea/chatqna --set global.HUGGINGFACEHUB_API_TOKEN=${HFTOKEN}
+```
+Verify deployment status:
+- Deployed Kubernetes services:
+
+  These are the actual services running in the cluster. Use the following command to list their runtime names:
+  ```bash
+  kubectl get svc
+  ```
+- Check the status of pods and ensure they are ready before proceeding:
+  ```bash
+  kubectl get pods
+  ```
+
+#### 2. Deploy with Gaudi Devices
+
+To offload TEI embedding and TGI inference to Intel Gaudi devices, use the specific values file.
+
+```bash
+# Extract chart contents to access provided configuration files.
+helm pull opea/chatqna --untar
+
+# Deploy using the Gaudi-specific configuration.
+helm install chatqna opea/chatqna --set global.HUGGINGFACEHUB_API_TOKEN=${HFTOKEN} -f chatqna/gaudi-values.yaml 
+
+```
+
+#### 3. Deploy with Multiple Instances
+
+To effectively deploy your workload across multiple nodes, follow these steps.
+
+##### Preload Shared Models
+Donwloading models simultaneously across multiple instances can overload resources. To prevent resource exhaustion, it's recommended to preload the models in advance.
```

Comment:
```suggestion
Downloading models simultaneously to multiple nodes in your cluster can overload network resources. To prevent resource exhaustion, it's recommended to preload the models to a local cache and re-use as illustrated below.
```

Reply:
Changed as "Downloading models simultaneously to multiple nodes in your cluster can overload resources such as network bandwidth, memory and storage. To prevent resource exhaustion, it's recommended to preload the models in advance."